tool/dev/install_app_version

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euxo pipefail
+
+app="$1"
+url="$2"
+
+cd /usr/src/nextcloud
+
+curl -Lo "$app.tar.gz" "$url"
+tar -xzf "$app.tar.gz" -C "apps"
+rm -r "$app.tar.gz"
```

Comment:
Can you just pipe the output of curl into tar.
Would avoid needing to write to disk.